tests/MongoDB.Driver.Tests/Specifications/server-discovery-and-monitoring/ServerDiscoveryAndMonitoringIntegrationTestRunner.cs

```diff
@@ -30,21 +31,25 @@
 
 namespace MongoDB.Driver.Tests.Specifications.server_discovery_and_monitoring
 {
-    public class ServerDiscoveryAndMonitoringIntegrationTestRunner : DisposableJsonDrivenTestRunner, IJsonDrivenTestRunner
+    public class ServerDiscoveryAndMonitoringIntegrationTestRunner : MongoClientJsonDrivenTestRunnerBase, IJsonDrivenTestRunner
     {
+        private readonly List<IDisposable> _disposables = new List<IDisposable>();
+
+        public IMongoClient FailPointClient => DriverTestConfiguration.Client;
```

Comment:
For some reason, some tests in this test suite fail with `MultipleShardRouters` connectionstring in sharded server. So, I added only regular test client here